examples/mobile/HomeApp/index.html

```diff
@@ -178,6 +183,26 @@ <h2>Channel 3</h2>
 					</li>
 				</ul>
 			</div>
+			<div class="ui-popup" id="turn-on-dialog">
+				<div class="ui-popup-header">
+					Add new widget
```

Comment:
I think "Set additional widgets" has more sense as a title.
This is because if additional widget is already set before this window shows checkbox for it